pkg/pillar/cmd/tpmmgr/tpmmgr.go

```diff
@@ -68,9 +68,10 @@ const (
 	//TpmDeviceCertHdl is the well known TPM NVIndex for device cert
 	TpmDeviceCertHdl tpmutil.Handle = 0x1500000
 
-	emptyPassword = ""
-	tpmLockName   = types.TmpDirname + "/tpm.lock"
-	maxPCRIndex   = 23
+	emptyPassword    = ""
+	tpmLockName      = types.TmpDirname + "/tpm.lock"
+	maxPCRIndex      = 23
+	tpmDoneTouchFile = "/var/run/tpm_done.touch"
```

Comment:
I think we should generalize the done pattern to have the zedbox code touch agentname.done when the goroutine returns.

Reply:
Done.